packages/emitter-framework/src/testing/scenario-test/harness.ts

```diff
@@ -62,6 +63,10 @@ async function assertGetEmittedFile(
   return sourceFile;
 }
 
+function normalizePath(path: string) {
```

Comment:
we've been using `pathe` in a bunch of places(and the compiler has its own path-utils) that just treat path always as unix style and hide the evil `\`

Reply:
Thanks! I'll import normalizePath from the compiler :)